### PR TITLE
1579 document feat implement alter databaseschema xxx rename to xxx

### LINKED ISSUE #1579: Document: feat: Implement `ALTER DATABASE/SCHEMA xxx RENAME TO xxx`
This issue tracks the documentation update needed for the merged PR #13713.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/13713
Source PR Merged At: 2023-11-30T02:20:22Z

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - There are mainly two aspects of changes: 
  - 1. Add RENAME TO to many commands according to the code PR and #7745  mentioned in this code PR. The table below shows the details:

| Command            | Add the command | Add RENAME TO | Status |
| ------------------ | --------------- | ------------- | ------ |
| ALTER TABLE        |                 | Y             | done   |
| ALTER MATERIALIZED |                 | Y             | done   |
| ALTER VIEW         |                 | Y             | done   |
| ALTER SINK         |                 | Y             | done   |
| ALTER INDEX        | Y               | Y             | done   |
| alter schema       |                 | Y             | done   |
| alter database     |                 | Y             | done   |

- 2. Adjust the document structure slightly to reduce repetition, making it easier for future document updates. Modifies some small details to achieve content consistency. The documentation structure of the command which has many clauses is now like this: 

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/13713 and [#7745](https://github.com/risingwavelabs/risingwave/pull/7745)

- **Related doc issue**
  
  Resolves [ https://github.com/risingwavelabs/risingwave-docs/issues/1579#event-11104661323 ]

## For reviewers

- **Preview**

  - [ https://pr-1590.d2fbku9n2b6wde.amplifyapp.com/docs/current/intro/]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/sql/commands/sql-alter-connection.md [hunk 1]
```diff
@@ -21,29 +17,20 @@ ALTER CONNECTION connection_name
...
 ```
 
-*`alter_option`* depends on the operation you want to perform on the connection.
+*`alter_option`* depends on the operation you want to perform on the connection. For all supported clauses, see the sections below.
 
-```sql
-ALTER CONNECTION connection_name
-    SET SCHEMA schema_name
-```
+## Clause
```
Comment (hengm3467) [inline docs/sql/commands/sql-alter-connection.md:22]: @ShanlanLi  Let's use plural even if there is only one clause in some cases.

Comment (hengm3467) [reply docs/sql/commands/sql-alter-connection.md:22]: This comment applies to all headings titled "Clause".

**Review by hengm3467 — APPROVED** [review]
Otherwise looks good to me. Let's make the minor change and merge as soon as possible.

File: docs/sql/commands/sql-alter-connection.md (hunk 1 above)
Comment (ShanlanLi) [reply docs/sql/commands/sql-alter-connection.md:22]: @hengm3467  But our other headings are singular, like Example, Syntax, like this: https://docs.risingwave.com/docs/dev/sql-cancel-jobs/. Do I need to change them too?

Comment (ShanlanLi) [reply docs/sql/commands/sql-alter-connection.md:22]: Thanks for the suggestions, after making changes in this PR according to our meeting last week, I don't plan to make changes except for solving doc issues from now on.

Comment (hengm3467) [reply docs/sql/commands/sql-alter-connection.md:22]: @ShanlanLi Good catch. Yes. Let's use plural for similar headings when possible.

**Pull request merged**